aerospike/hatch.toml

```diff
@@ -0,0 +1,21 @@
+[env.collectors.datadog-checks]
+check-types = true
+mypy-args = [
+    "--py2",
+    "--follow-imports silent",
+    "--install-types",
+    "--non-interactive",
+    "datadog_checks/aerospike",
+    "--exclude '.*/config_models/.*\\.py$'",
+]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+aerospike-exporter-version = ["1.5.0"]
```

Comment:
This seems like we're only pinning, so let's keep it as an env var https://hatch.pypa.io/latest/config/environment/overview/#defined